proto/kaskada/kaskada/v1alpha/sources.proto

```diff
@@ -2,12 +2,14 @@ syntax = "proto3";
 
```

Comment:
I would have expected to see a similar update to `destinations.proto`

Reply:
I'm aiming for read only from kafka first and destinations can come after.